gadgets/trace_bind/gadget.yaml

```diff
@@ -56,9 +56,9 @@ datasources:
           columns.width: 10
 ebpfParams:
   ignore_errors:
-    key: ignore_errors
+    key: ignore-errors
     alias: i
-    defaultValue: ""
+    defaultValue: "true"
```

Comment:
note: It's actually not needed anymore as we're getting these from the ebpf object. (I'll open an issue / PR to clean it up).